src/aphroditeInterfaceWithRTLFactory.js

```diff
@@ -0,0 +1,25 @@
+import { flushToStyleTag } from 'aphrodite/lib/inject';
+
+import resolveWithRTL from './utils/resolveWithRTL';
+import resolve from './utils/resolve';
+
+export default ({ StyleSheet, css }/* aphrodite */) => ({
```

Comment:
why the comment?

Reply:
This already existed and is there to clarify that the way you accomplish this is by passing in aphrodite itself I think.

... I did not add this comment.